api/queries/catalog.queries.js

```diff
@@ -21,6 +21,10 @@ export const catalogBySeasonQuery = gql`
       average_professor
       average_rating
       average_workload
+      course {
+        average_rating_same_professors
+        average_workload_same_professors
+      }
```

Comment:
@kevinhu can we get these copied into the generated table

Reply:
Done! FYI @maxyuan6717 we now have columns indicating the number of classes used to calculate ratings (just the name with `_n` at the end).